Formula/d/dstack.rb

```diff
@@ -451,6 +450,11 @@ class Dstack < Formula
     sha256 "89075171ef11988b3fa157f5dbd8b9cf09d65fffee97e29ce403cd8defba19d2"
   end
 
+  resource "protobuf" do
+    url "https://files.pythonhosted.org/packages/5e/d8/65adb47d921ce828ba319d6587aa8758da022de509c3862a70177a958844/protobuf-4.25.3.tar.gz"
+    sha256 "25b5d0b42fd000320bd7830b349e3b696435f3b329810427a6bcce6a5492cc5c"
+  end
```

Comment:
Does this end up building its own protobuf binaries/libraries? If so we may want to convince it to use our own.

Reply:
The libraries are probably different given upstream deprecated using C++ backend - https://github.com/protocolbuffers/protobuf/blob/main/python/README.md

According to https://github.com/protocolbuffers/protobuf/tree/main/upb
> While upb offers a C API, the C API & ABI are not stable. For this reason, upb is not generally offered as a C library for direct consumption, and there are no releases.